pkg/tanka/prune.go

```diff
@@ -15,11 +16,22 @@ type PruneOpts struct {
 	AutoApprove bool
 	// Force ignores any warnings kubectl might have
 	Force bool
+	// Also prune namespaces
+	IncludeNamespaces bool
 }
 
 // Prune deletes all resources from the cluster, that are no longer present in
 // Jsonnet. It uses the `tanka.dev/environment` label to identify those.
 func Prune(baseDir string, opts PruneOpts) error {
+	// By default, don't delete namespaces
+	if !opts.IncludeNamespaces {
+		ignoreNamespace, err := process.StrExps("!namespace*")
```

Comment:
```suggestion
		ignoreNamespace, err := process.StrExps("!namespace/.+")
```

This is a regex, so your quantifier `*` needs an according wildcard (e.g. `.`). Your current expression only matches inputs of form `namespaceX` with X being any single char.
Also it's good to include the `/` so that only kind `namespace` but not other kinds that start with namespace are matched

Reply:
(read my other comment, this makes no longer sense)